internal/blocksync/reactor.go

```diff
@@ -537,9 +537,9 @@ FOR_LOOP:
 			blocksSynced++
 
 			if blocksSynced%100 == 0 {
+				_, height, maxPeerHeight := bcR.pool.IsCaughtUp()
 				lastRate = 0.9*lastRate + 0.1*(100/time.Since(lastHundred).Seconds())
```

Comment:
Do you know why we use this window mechanism?

Reply:
I was meaning the 90% of the last rate + 10% of the current. This is a form of moving window, I don't remember the precise name of this method. Anyway, just a comment.